packages/test-framework/src/fixture/pagePreset/PaymentStepAsGuest.ts

```diff
@@ -9,14 +9,15 @@ export class PaymentStepAsGuest implements CheckoutPagePreset {
         private currency: string = 'USD',
         private countryCode?: string,
         private locale: string = Locales.US,
+        private postalCode?: string,
     ) {}
     async apply(page: Page): Promise<void> {
         const api = new ApiRequestsSender(page, this.locale);
 
         await api.addPhysicalItemToCart();
         await api.setCurrency(this.currency);
         await api.completeCustomerStepAsGuest();
-        await api.completeSingleShippingAndSkipToPaymentStep(this.countryCode);
+        await api.completeSingleShippingAndSkipToPaymentStep(this.countryCode, this.postalCode);
```

Comment:
Why the change here?

Reply:
Removed. This code is no more needed because I mocked Stripe config.